src/server/AccountsServer.js

```diff
@@ -4,9 +4,21 @@ import { defaultServerConfig } from '../common/defaultConfigs';
 import { AccountsError } from '../common/errors';
 import AccountsCommon from '../common/AccountsCommon';
 import type { AccountsOptionsType } from '../common/AccountsCommon';
+import type { DBDriverInterface } from './DBDriver';
+import type UserObjectType from '../common/UserObjectType';
+
+
+export type UserCreationInputType = {
```

Comment:
Passwords are optional on the server but they are not optional on the client so we need to have two different input types.